BDEPEND=test? ( dev-python/zope-interface[python_targets_pypy3(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,python_targets_python3_10(-)?] dev-python/unittest-or-fail[python_targets_pypy3(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,python_targets_python3_10(-)?] ) python_targets_pypy3? ( >=dev-python/pypy3-7.3.7-r1:0= ) python_targets_python3_8? ( >=dev-lang/python-3.8.12_p1-r1:3.8 ) python_targets_python3_9? ( >=dev-lang/python-3.9.9-r1:3.9 ) python_targets_python3_10? ( >=dev-lang/python-3.10.0_p1-r1:3.10 ) >=dev-python/installer-0.4.0_p20220124[python_targets_pypy3(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,python_targets_python3_10(-)?] >=dev-python/tomli-1.2.3[python_targets_pypy3(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,python_targets_python3_10(-)?] >=dev-python/setuptools-60.5.0[python_targets_pypy3(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,python_targets_python3_10(-)?] dev-python/wheel[python_targets_pypy3(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,python_targets_python3_10(-)?]
DEFINED_PHASES=compile configure install prepare test
DESCRIPTION=General purpose exceptions for Zope packages
EAPI=8
HOMEPAGE=https://pypi.org/project/zope.exceptions/ https://github.com/zopefoundation/zope.exceptions/
IUSE=test python_targets_pypy3 python_targets_python3_8 python_targets_python3_9 python_targets_python3_10
KEYWORDS=~amd64 ~arm ~arm64 ~ppc64 ~x86
LICENSE=ZPL
RDEPEND=dev-python/zope-interface[python_targets_pypy3(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,python_targets_python3_10(-)?] python_targets_pypy3? ( >=dev-python/pypy3-7.3.7-r1:0= ) python_targets_python3_8? ( >=dev-lang/python-3.8.12_p1-r1:3.8 ) python_targets_python3_9? ( >=dev-lang/python-3.9.9-r1:3.9 ) python_targets_python3_10? ( >=dev-lang/python-3.10.0_p1-r1:3.10 )
REQUIRED_USE=|| ( python_targets_pypy3 python_targets_python3_8 python_targets_python3_9 python_targets_python3_10 )
RESTRICT=!test? ( test )
SLOT=0
SRC_URI=mirror://pypi/z/zope.exceptions/zope.exceptions-4.5.tar.gz
_eclasses_=distutils-r1	779bc733c68a72273cfce64602bada1d	multibuild	d26d81f242cb193d899a72bca423d0bd	multilib	4a33c9008e5ee30cb8840a3fdc24df2b	multiprocessing	61c959fc55c15c00bbb1079d6a71370b	python-r1	8638751691200e941f26fe0ac3aef1d1	python-utils-r1	59dddabd6a41ab69723654916142f63b	toolchain-funcs	badd6e329e1f3e6bee99b35bf8763ce8
_md5_=141df81d52f7a5abdcf7dd4748d59dce
